lighthouse-core/audits/font-display.js

```diff
@@ -57,7 +57,9 @@ class FontDisplay extends Audit {
       const fontFaceDeclarations = newlinesStripped.match(/@font-face\s*{(.*?)}/g) || [];
       // Go through all the @font-face declarations to find a declared `font-display: ` property
       for (const declaration of fontFaceDeclarations) {
-        const rawFontDisplay = declaration.match(/font-display:(.*?);/);
+        // Find the font-display value by matching a single token, optionally surrounded by whitespace,
+        // followed either by a semicolon or the end of a block.
+        const rawFontDisplay = declaration.match(/font-display:(\s*\S+\s*)(;|\s*\})/);
```

Comment:
is the last `\s*` not already captured by the first group?

Reply:
after just getting burned by the old regex, I feel like we need some extra whitespace tests for the cases we're going to be handling now :)

Reply:
+1 to @brendankenny first comment, should be:
```
font-display:(\s*\S+\s*)(;|\})
```
iirc it is valid to have white space after the property before the colon like:
```css
font-display : 'optional';
```
in which case we would need:
```
font-display\s*:(\s*\S+\s*)(;|\})
```